.golangci.yml

```diff
@@ -21,6 +21,17 @@ linters-settings:
       # Replace these for trivy
       - github.com/docker/docker
       - oras.land/oras-go
+      # End-to-end testing
+      - github.com/spdx/tools-golang
+      - go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc
+      - go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp
+      - go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracehttp
+      - cloud.google.com/go
+      - github.com/cucumber/godog
+      - k8s.io/api
+      - k8s.io/apimachinery
+      - k8s.io/client-go
+      - github.com/docker/cli
```

Comment:
Do we really need all the replaces in the e2e? Especially the `k8s.io` ones?

Reply:
@FrimIdan Sadly, we do. All of them are coming from docker.

Reply:
Do you think we should have a different golangci configuration for the e2e?